src/Builder/PayloadBuilder.php

```diff
@@ -18,6 +18,8 @@ public function __construct(
     ) {
         if ($dimensions) {
             collect($dimensions)->each(fn ($value, $key) => $this->buildDimensions($key, $value));
+        } else {
```

Comment:
Instead of adding an else statement to default it to an empty array,

can you rather initialize it on line 10 where I declare the variable?

```php
protected array $dimensions = [];
```

by doing the above we won't need the else statement anymore.

Reply:
Thanks for your comment. 

Done. :)